src/qcodes/dataset/dond/do_1d.py

```diff
@@ -74,106 +42,28 @@ def do1d(
         stop: End point of sweep
         num_points: Number of points in sweep
         delay: Delay after setting parameter before measurement is performed
-        param_meas: Parameter(s) to measure at each step or functions that
+        *param_meas: Parameter(s) to measure at each step or functions that
           will be called at each step. The function should take no arguments.
           The parameters and functions are called in the order they are
           supplied.
-        enter_actions: A list of functions taking no arguments that will be
-            called before the measurements start
-        exit_actions: A list of functions taking no arguments that will be
-            called after the measurements ends
-        write_period: The time after which the data is actually written to the
-            database.
-        additional_setpoints: A list of setpoint parameters to be registered in
-            the measurement but not scanned.
-        measurement_name: Name of the measurement. This will be passed down to
-            the dataset produced by the measurement. If not given, a default
-            value of 'results' is used for the dataset.
-        exp: The experiment to use for this measurement.
-        do_plot: should png and pdf versions of the images be saved after the
-            run. If None the setting will be read from ``qcodesrc.json``
-        use_threads: If True measurements from each instrument will be done on
-            separate threads. If you are measuring from several instruments
-            this may give a significant speedup.
-        show_progress: should a progress bar be displayed during the
-            measurement. If None the setting will be read from ``qcodesrc.json``
-        log_info: Message that is logged during the measurement. If None a default
-            message is used.
-        break_condition: Callable that takes no arguments. If returned True,
-            measurement is interrupted.
+        **kwargs: kwargs are the same as for dond and forwarded directly to dond.
```

Comment:
perhaps use fancy sphinx references so that users can get to documentation of these arguemtns in hte docs quickly via a link ?
```suggestion
        **kwargs: kwargs are the same as for :func:`dond` and forwarded directly to :func:`dond`.
```